packages/components/src/molecules/Table/Table.tsx

```diff
@@ -14,6 +18,9 @@ export interface TableProps extends TableHTMLAttributes<HTMLTableElement> {
    * Defines what style this component should use.
    */
   variant?: 'colored' | 'bordered'
+  onResize?: ReactEventHandler<unknown> | undefined
+  onResizeCapture?: ReactEventHandler<unknown> | undefined
+  nonce?: string | undefined
```

Comment:
Maybe add a description for each prop here?

Reply:
done [here](https://github.com/vtex/faststore/pull/1601/commits/5048fca908dee03152f5ded5c64fba5a68c9640a).